x-pack/plugins/security_solution/cypress/integration/overview/cti_link_panel.spec.ts

```diff
@@ -0,0 +1,170 @@
+      cy.get(`${OVERVIEW_CTI_LINKS} ${OVERVIEW_CTI_LINKS_ERROR_INNER_PANEL}`).should('exist');
+    });
+
+    it('renders disabled "view dashboard" button', () => {
+      cy.get(`${OVERVIEW_CTI_VIEW_DASHBOARD_BUTTON}`).should('be.disabled');
+    });
+
+    it('renders 0 total event count', () => {
+      cy.get(`${OVERVIEW_CTI_TOTAL_EVENT_COUNT}`).should('have.text', 'Showing: 0 events');
+    });
+
+    it('renders enable module button', () => {
+      it('renders disabled "view dashboard" button', () => {
+        cy.get(`${OVERVIEW_CTI_ENABLE_MODULE_BUTTON}`).should('exist');
+        // TODO: how to test injected doclink?
+        // cy.get(`${OVERVIEW_CTI_ENABLE_MODULE_BUTTON} a`).should('have.attr', 'href', "doclink");
+      });
+    });
+  });
+
+  describe('enabled threat intel module', () => {
+    before(() => {
+      esArchiverLoad('threat_indicator');
+    });
+
+    after(() => {
+      esArchiverUnload('threat_indicator');
+    });
+
+    describe('disabled dashboard module', () => {
+      describe('when there are no events in the selected time period', () => {
+        before(() => {
+          loginAndWaitForPage(
+            `${OVERVIEW_URL}?sourcerer=(timerange:(from:%272021-07-08T04:00:00.000Z%27,kind:absolute,to:%272021-07-09T03:59:59.999Z%27))`
+          );
+        });
+
+        it('renders warning inner panel', () => {
+          cy.get(`${OVERVIEW_CTI_LINKS} ${OVERVIEW_CTI_LINKS_WARNING_INNER_PANEL}`).should('exist');
+        });
+
+        it('renders info inner panel', () => {
+          cy.get(`${OVERVIEW_CTI_LINKS} ${OVERVIEW_CTI_LINKS_INFO_INNER_PANEL}`).should('exist');
+        });
+
+        it('renders disabled "view dashboard" button', () => {
+          cy.get(`${OVERVIEW_CTI_VIEW_DASHBOARD_BUTTON}`).should('be.disabled');
+        });
+
+        it('renders 0 total event count', () => {
+          cy.get(`${OVERVIEW_CTI_TOTAL_EVENT_COUNT}`).should('have.text', 'Showing: 0 events');
+        });
+      });
+
+      describe('when there are events in the selected time period', () => {
+        before(() => {
+          loginAndWaitForPage(OVERVIEW_URL);
+        });
+
+        it('does not render warning inner panel', () => {
+          cy.get(`${OVERVIEW_CTI_LINKS} ${OVERVIEW_CTI_LINKS_WARNING_INNER_PANEL}`).should(
+            'not.exist'
+          );
+        });
+
+        it('renders info inner panel', () => {
+          cy.get(`${OVERVIEW_CTI_LINKS} ${OVERVIEW_CTI_LINKS_INFO_INNER_PANEL}`).should('exist');
+        });
+
+        it('renders disabled "view dashboard" button', () => {
+          cy.get(`${OVERVIEW_CTI_VIEW_DASHBOARD_BUTTON}`).should('be.disabled');
+        });
+
+        it('renders correct total event count', () => {
+          cy.get(`${OVERVIEW_CTI_TOTAL_EVENT_COUNT}`).should('have.text', 'Showing: 1 event');
+        });
+      });
+    });
+
+    // TODO: find a way to properly load a 'threat intel' tag, and a dashboard that has that tag, for the following tests to work
```

Comment:
I know that persisting saved objects in archives has historically been discouraged, as they can quickly become outdated as migrations are released.

You could potentially issue an API call to create the necessary objects, here. If these were converted to FTR tests, you could also use the savedObjects client directly, as we do in [other FTR tests](https://github.com/elastic/kibana/blob/master/x-pack/test/detection_engine_api_integration/security_and_spaces/tests/create_signals_migrations.ts#L60-L63)

Reply:
issue tracked here elastic/security-team#1438

Reply:
Can we delete these TODOs and archive for now, then?